src/exchanges/bitfinex-client.js

```diff
@@ -129,7 +144,6 @@ class BitfinexClient extends BasicClient {
 
   _onMessage(raw) {
     let msg = JSON.parse(raw);
-
```

Comment:
nit: mind your spaces haha

Reply:
I'm gunna keep doing these by mistake lol so I really recommend getting a custom eslint config for these rules since you're using prettier but also have some other rules in your mind as well, you'd save yourself a lot headache. using a simple pre-commit and pre-push git hook that uses a custom eslint config and rejects improperly formatted code from even being pushed up to the repo would do the trick

Reply:
Yeah, its on the todo list.  In the interim, just try to keep with the flow.  Stuff like this nit is obvious in the diff.